.github/workflows/test.yml

```diff
@@ -206,7 +206,7 @@ jobs:
         if: "env.GIT_DIFF != ''"
   liveness-test:
     runs-on: ubuntu-latest
-    timeout-minutes: 10
+    timeout-minutes: 20
```

Comment:
why the increase by so much?

Reply:
Probably this was a test by @sahith-narahari , we can revert this back.

Reply:
```suggestion
    timeout-minutes: 10
```